Example/Tests/Tests.m

```diff
@@ -360,5 +385,42 @@
         });
     });
 
+    describe(@"video tracking", ^{
+        describe(@"initialization", ^{
+            __block ADBMediaHeartbeat *mockADBMediaHeartbeat;
+            ADBMediaHeartbeatConfig *config = [[ADBMediaHeartbeatConfig alloc] init];
+
+            beforeEach(^{
+                mockADBMediaHeartbeat = mock([ADBMediaHeartbeat class]);
+                SEGMockADBMediaHeartbeatFactory *mockADBMediaHeartbeatFactory = [[SEGMockADBMediaHeartbeatFactory alloc] init];
+                mockADBMediaHeartbeatFactory.ADBMediaHeartbeat = mockADBMediaHeartbeat;
+                integration = [[SEGAdobeIntegration alloc] initWithSettings:@{ @"heartbeatTrackingServer" : @"example" }
+                                                               andADBMobile:mockADBMobile
+                                                andADBMediaHeartbeatFactory:mockADBMediaHeartbeatFactory
+                                                 andADBMediaHeartbeatConfig:config];
+            });
+
+
+            it(@"Video Playback Started initializes ADBMediaHeartbeat object", ^{
+                SEGTrackPayload *payload = [[SEGTrackPayload alloc] initWithEvent:@"Video Playback Started" properties:@{
+                    @"content_asset_id" : @"1234",
+                    @"ad_type" : @"pre-roll",
+                    @"video_player" : @"Netflix",
+                    @"channel" : @"Cartoon Network",
+                    @"" : @""
+                } context:@{}
+                                                                     integrations:@{ @"Adobe Analytics" : @{@"ovpName" : @"Netflix", @"debug" : @YES} }];
```

Comment:
@f2prateek you mentioned  "don't think this would be an integration specific option as it's used in initialization," however, since initialization occurs on `Video Playback Started`, users will have the opportunity to pass this in. 

Happy to hear what you had in mind